src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -415,7 +415,10 @@ class certificates extends viewModelBase {
                 const serverCert = mergedCertificates.find(x => _.includes(x.Thumbprints, this.serverCertificateThumbprint()));
                 const clientCert = mergedCertificates.find(x => _.includes(x.Thumbprints, this.clientCertificateThumbprint()));
                 const orderedCertificates = mergedCertificates.filter(x => x !== serverCert && x !== clientCert);
-                orderedCertificates.unshift(clientCert);
+                
+                if(!!clientCert) {
```

Comment:
please format code and no need to use !! in condition

Reply:
right - done